common/dao-api/src/main/java/org/thingsboard/server/dao/entityprofile/EntityProfileService.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright © 2016-2020 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.dao.entityprofile;
+
+import org.thingsboard.server.common.data.entityprofile.EntityProfile;
+import org.thingsboard.server.common.data.id.EntityProfileId;
+import org.thingsboard.server.common.data.id.TenantId;
+
+import java.util.List;
+
+public interface EntityProfileService {
+
+    List<EntityProfile> findAll(TenantId tenantId);
```

Comment:
We need ability to find EntityProfile using EntityType. 

```java
PageData<EntityProfile> findEntityProfilesByTenantId(TenantId tenantId, PageLink pageLink);
PageData<EntityProfile> findEntityProfilesByTenantIdAndType(TenantId tenantId, EntityType entityType, PageLink pageLink);
// This will be used by system administrator. TenantId is SYS_TENANT_ID in this case and EntityType = Tenant
PageData<EntityProfile> findTenantProfiles(PageLink pageLink);
```

Reply:
added, @ashvayka can you check one more time? 
if I understand correct, findTenantProfiles can be default method

Reply:
This is ok, but findAll needs to be removed. There might be a lot of profiles..